docs/gateway-flows.md

```diff
@@ -1,16 +1,20 @@
 # Flow diagrams for common Gateway routes
 
-These flow diagrams are a WIP, mostly used to help visualise some complex functions in Gateway for the purposes of mocking Okta flows in testing.
+**_Note: These diagrams are outdated and describe behaviour for the older Okta Classic API. For more up to date flows using the newer Okta IDX API, see the [IDX Documentation](./okta/idx/README.md)_**
```

Comment:
if they are outdated why do we keep them ?

Reply:
Because the code hasn't been removed, and can still be accessed if using `useOktaClassic=true` or if using tokens from the Classic Authentication API which is still used in some cases, i.e activation emails from the guest user endpoint.